### ISSUE TITLE
Styles applied to non-applicable websites with an <iframe> from another domain

### ISSUE THREAD

**Issue by lawrence-m** [350d/stylish#37]
With the latest version (2.0.4), and Safari (11.0.1) or Safari Technology Preview (Release 43), styles for websites with different URLs are being applied when there is an embedded iframe for another domain on that site.

For example, `theguardian.com` has an iframe embed for `youtube.com`. When a style is applied to `youtube.com`, it will also be applied to `theguardian.com`.

![screen shot 2017-11-14 at 13 55 00](https://user-images.githubusercontent.com/9520041/32780966-747a7172-c943-11e7-9b26-801628d05b0f.PNG)
![screen shot 2017-11-14 at 13 54 54](https://user-images.githubusercontent.com/9520041/32780967-766b0410-c943-11e7-9b01-a4b34ab23e35.PNG)

**Comment by tzm41** [comment]
Yep, confirm that when I have a dark Google style, basically every website becomes dark...

**Comment by 350d** [comment]
#34 Another bug fix was a reason for this one.
Confirmed and updated in 2.0.6. Will be available in autoupdate soon or you can download it directly from website https://sobolev.us/stylish

**Comment by tzm41** [comment]
Thanks for fixing it!

**Comment by tzm41** [comment]
Not seeing the update though Safari Extension Gallery yet?

**Comment by tzm41** [comment]
Is 2.0.6 released?

**Comment by 350d** [comment]
2.0.8 released and submitted to Apple. Waiting for approval.

**Comment by tzm41** [comment]
Great! I am still on 2.0.5 and not seeing update yet.

**Comment by 350d** [comment]
@tzm41 I have beta release here, do you want to test it? (new editor available too)

**Comment by tzm41** [comment]
Ah sure

**Comment by 350d** [comment]
[stylish.2.0.9.zip](https://github.com/350d/stylish/files/1516482/stylish.2.0.9.zip)

**Comment by 350d** [comment]
New editor now supports EMMET shortcuts (ex ZEN Coding), so you still can expand any css shortcuts to full snippets with `TAB` key like `bgs:c` to `background-size: cover;`. Let me known what you think about it. CSS validation can show you the number of errors and highlight exact lines in realtime. Code highlighted with CodeMirror project.
